lib/src/json_schema/json_schema.dart

```diff
@@ -1071,9 +1074,18 @@ This functionality will be removed in 3.0.
     }
   }
 
+  /// Name of the property of the current [JsonSchema] within its parent.
+  String get propertyName {
+    final pathFragments = path.split('/');
```

Comment:
might be slightly safer to `Uri.parse(path).fragment?.split...`

Reply:
can't think of an explicit case where the path wouldn't be just a fragment, but just to be sure